app/controllers/api/v2/hosts_controller.rb

```diff
@@ -126,6 +126,13 @@ def destroy
         process_response @host.destroy
       end
 
+      api :GET, "/hosts/:id/enc", N_("Get ENC values of host")
+      param :id, :identifier_dottable, :required => true
+
+      def enc
+        render :json => { :data => @host.info }.to_json if @host
```

Comment:
I also though it could be redundant but as I saw the same checks in `status/` call I did it in the same way.

See line 151: it checks `if @host` and also applies `.to_json`.